Small fixes:

Astarsys opened the pull request:
- CAN state fields changed from boolean to strings.
- Better explanation for limit exceeded parameter in user events, that is related to user's tariff, not amount of events in response
- Deleted mobile-tracker.md as it is referring to 3rd party apps (plugins). The same as described in user apps.

fenik17 commented on docs/backend-api/resources/tracking/tracker/index.md:
```diff
@@ -460,24 +460,28 @@ List of state names for the field `states`:
-* **external_power_state** (connected/disconnected; value type: boolean).
-* **case_intrusion_state** (value type: boolean).
-* **driver_ident_state** (identified/not identified; value type: boolean).
+* **external_power_state** (connected/disconnected; value type: string).
```
Here we need a description of the possible values like you did it below. And what about `vibration_state`, `idling_state`? 
In fact, this is far from a complete list of possible state fields..

Astarsys replied:
Sure, I can add them all. Is there a way I can get information about all possible state fields and values? I added only for state fields I was confident the value is string and its meaning.

Astarsys replied:
added link to sensor/input_name/list API call with all descriptions.

Astarsys commented:
Also updated
- CAN state fields changed from boolean to strings. Added that state fields and other inputs may be requested with tracker/sensor/input_name/list API call
- Better explanation for every API call, parameter, example, response field and errors for track API calls.
- Deleted mobile-tracker.md as it is referring to 3rd party apps (plugins). The same as described in user apps.
- Changed information about the panel hash lifespan.
- removed Academy link from the API documentation.
- Better description of sensor/input_name/list API call.
